rubocop_default.yml

```diff
@@ -1,13 +1,9 @@
-require: 
+require:
   - rubocop-performance
   - rubocop-rspec
 
-inherit_from: 
+inherit_from:
   - rubocop_all.yml
-  - rubocop_layout.yml
-  - rubocop_lint.yml
-  - rubocop_metrics.yml
   - rubocop_rails.yml
-  - rubocop_style.yml
   - rubocop_performance.yml
```

Comment:
I left performance — didn't actually really know about the `rubocop-performance` gem. Had a look and it's super interesting. I've left the rules in since I am unfamiliar with it.